external/fv3fit/fv3fit/reservoir/autoencoder.py

```diff
@@ -35,11 +63,206 @@ def load(cls, path: str) -> "Autoencoder":
             decoder = tf.keras.models.load_model(
                 os.path.join(model_path, cls._DECODER_NAME)
             )
+
         return cls(encoder=encoder, decoder=decoder)
 
+
+@io.register("keras-autoencoder")
+class PureKerasAutoencoderModel(PureKerasModel):
```

Comment:
There isn't much in the way of functionality that we're adopting from the base class here to justify the cognitive cost of having inheritance. Since we weren't able to just use PureKerasModel, could you remove PureKerasModel as a parent class and copy any relevant code into this class instead?

Reply:
As discussed in slack, noted in issue https://github.com/ai2cm/fv3net/issues/2198